woocommerce/woocommerce-android
# [Woo POS] Empty state for products

Closes: #11881


### Description

Adds simple empty state for products screen

### Steps to reproduce


### Testing information

* Use a store without products
* Notice an empty state

### Images/gif


<img width="777" alt="image" src="https://github.com/woocommerce/woocommerce-android/assets/4923871/f4a6169e-c4dd-4abf-b10a-b7e440f25f97">
